config/default/kustomization.yaml

```diff
@@ -39,7 +39,7 @@ resources:
 - ../crd
 - ../rbac
 - ../manager
-- ../prometheus
-- ../clusteroperator
+#- ../prometheus
+#- ../clusteroperator
```

Comment:
Lets just remove them instead of commenting them?
And keep it in the new `default-ocp` directory?

Reply:
We can remove ` #- ../clusteroperator ` indeed. Not sure about Prometheus - Kubernetes users might still be interested in them.
What would a `default-ocp` directory bring?